src/datasources/data-frame/components/v2/DataFrameQueryEditorV2.tsx

```diff
@@ -156,6 +160,46 @@ export const DataFrameQueryEditorV2: React.FC<Props> = ({ query, onChange, onRun
 
                 </Collapse>
             </div>
+
+            {query.type === DataFrameQueryType.Data && (
+                <div
+                    style={{ width: getValuesInPixels(sectionWidth) }}
+                >
+                    <Collapse
+                        label={labels.columnConfigurations}
+                        isOpen={isColumnConfigurationSectionOpen}
+                        collapsible={true}
+                        onToggle={() => setIsColumnConfigurationSectionOpen(!isColumnConfigurationSectionOpen)}
+                    >
+                        <InlineField
+                            label={labels.columns}
+                            labelWidth={inlineLabelWidth}
+                            tooltip={tooltips.columns}
+                        >
+                            <MultiSelect
+                                onChange={onColumnChange}
+                                options={[]}
+                                allowCustomValue={false}
+                            />
+                        </InlineField>
+                        <InlineField
+                            label={labels.filterNulls}
+                            labelWidth={inlineLabelWidth}
+                            tooltip={tooltips.filterNulls}>
```

Comment:
```suggestion
                        <InlineField
                            label={labels.filterNulls}
                            labelWidth={inlineLabelWidth}
                            tooltip={tooltips.filterNulls}
                        >
```
nit: update in other places.

I feel this would improve readbility.

Reply:
updated